NuRadioReco/framework/station.py

```
@@ -261,6 +261,36 @@ def get_magnetic_field_vector(self, time=None):
             logger.error(f"Reference reconstruction not set / unknown: {self.__reference_reconstruction}")
             raise ValueError(f"Reference reconstruction not set / unknown: {self.__reference_reconstruction}")
 
+    def avg_SNR(self, channels_to_include = [0, 1, 2, 3, 5, 6, 7, 22, 23]):
+        """
+        Returns the average SNR across all channels
+        """
+        snr_all = []
+        for channel in station.iter_channels():
+            if (channel.has_parameter(chp.SNR)):
+                snr_ch = channel.get_parameter(chp.SNR)
+                if (snr_ch == np.inf):
+                    return np.inf
+                else:
+                    snr_all.append(snr_ch)
```

@avijai2000, I think currently `channels_to_include` is not used. You can pass it to `station.iter_channels(channels_to_include)`. Can you also rename the function to something like `get_average_signal_to_noise_ratio` (at least I am a fan or being quite explicit). Furthermore, I would probably raise and error (or warning) if the channel parameter are missing). Also I think the enums are not wrong (at least in case of the RPR). All comments also apply to the RPR function

---

Just made the changes! Let me know what you think